src/routes/lobby/LobbyView.vue

```diff
@@ -151,6 +174,12 @@ export default {
       }
     },
   },
+  async created() {
+    if (!this.gameStore.id || this.gameStore.id === null) {
+      this.subscribeToGame(window.location.hash.split('lobby/')[1]);
+      this.$router.push(`/lobby/${window.location.hash.split('lobby/')[1]}`);
+    }
+  },
```

Comment:
It feels weird to have a function where the entire thing is wrapped in an if, you should instead use a fast fail pattern here and invert the logic to return.